client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HubConnection.ts

```diff
@@ -48,7 +52,7 @@ export class HubConnection {
         var descriptor = JSON.parse(data);
         if (descriptor.Method === undefined) {
             let invocationResult: InvocationResultDescriptor = descriptor;
-            let callback = this.callbacks[invocationResult.Id];
+            let callback = this.callbacks.get(invocationResult.Id);
```

Comment:
Because it is strongly typed (in TS sense) and gives you intellisense in vscode.